Sources/SwiftFusion/Geometry/Pose3.swift

```diff
@@ -97,6 +105,14 @@ extension Pose3Coordinate: LieGroupCoordinate {
   }
 }
 
+extension Pose3Coordinate {
+  /// Group action on `Vector3`.
+  @differentiable
+  static func * (_ lhs: Pose3Coordinate, _ rhs: Vector3) -> Vector3 {
```

Comment:
So, I get why you would use `lhs` and `rhs`, but I would propose changing all these methods to use the "coordinate system memnonic", as extra documentation and sanity check. In GTSAM we write a pose as 'a_pose_b' where a refers to the target frame A and b to the source frame B. We then write `a_point = a_pose_b * b_point`, and you can see that indices cancel, as they would in tensor contraction @dabrahams :-)

Same for rotations, so below it would read as:
```
a_rotation_b * b_point + a_translation
```

Note also I *deliberately* choose Rotation/translation order, and *deliberately* don't abbreviate. I think of a 3D pose as [R,t], not [t,R]. Let's make our code think the same way :-)

Reply:
Yes, `b_point` makes it clearer that the point is currently in the source frame B and we want to move it to the target frame A. I don't actually have a hard opinion on this though. For me the semantics is already quite clear and unambiguous as it is.

So, what about something like this?
```swift
static func * (_ a_pose_b: Pose3Coordinate, _ b_point: Vector3) -> Vector3 {
    a_pose_b.rot * b_point + a_pose_b.t
}
```

Or should we go with a more verbose version?
```swift
static func * (_ a_pose_b: Pose3Coordinate, _ b_point: Vector3) -> Vector3 {
    let a_rotation_b = a_pose_b.rot
    let a_translation = a_pose_b.t
    return a_rotation_b * b_point + a_translation
}
```

Also, these break the naming convention rule, is it okay?

Btw, what is tensor contraction? Any primer on the topic to help me get started?

Reply:
If naming convention is camelCase, then let's use aTb, aRb etc. I do prefer the less verbose version above.

Reply:
Done.